docs/user/troubleshooting.md

```diff
@@ -244,13 +244,13 @@ If it doesn't exist, the installer didn't create it. You'll have to run `openshi
 Next, check that the network-operator is running:
 
 ```sh
-kubectl -n openshift-cluster-network-operator get pods
+kubectl -n openshift-network-operator get pods
 ```
 
 And retrieve the logs. Note that, on multi-master systems, the operator will perform leader election and all other operators will sleep:
 
 ```sh
-kubectl -n openshift-cluster-network-operator logs -l "k8s-app=cluster-network-operator"
+kubectl -n openshift-network-operator logs -l "name=network-operator"
```

Comment:
@danwinship can you confirm this change of label selector works. 😇

Reply:
```
danw@p50:~> kubectl -n openshift-network-operator logs -l "name=network-operator" | tail -5
2019/01/29 17:16:48 reconciling (/v1, Kind=ConfigMap) openshift-sdn/sdn-config
2019/01/29 17:16:48 update was noop
2019/01/29 17:16:48 reconciling (apps/v1, Kind=DaemonSet) openshift-sdn/sdn
2019/01/29 17:16:49 update was noop
2019/01/29 17:16:49 all objects successfully applied
```